src/traffic_server/InkAPITest.cc

```diff
@@ -83,6 +83,29 @@
 #define ERROR_BODY "TESTING ERROR PAGE"
 #define TRANSFORM_APPEND_STRING "This is a transformed response"
 
+//////////////////////////////////////////////////////////////////////////////
+// These static asserts verify the hook vs event value relationship described
+// in the doxygen comment for TSHttpHookID.
+//////////////////////////////////////////////////////////////////////////////
+static_assert(TS_EVENT_HTTP_READ_REQUEST_HDR == TS_EVENT_HTTP_READ_REQUEST_HDR + TS_HTTP_READ_REQUEST_HDR_HOOK);
```

Comment:
This is overkill. You want this, or the `REBASE` macro, not both.

Reply:
Yeah, I suppose if the macro is wrong, it will mess up all the event values and it will be pretty obvious when a bunch of autests fail.